docs/packages/pkg/opentracing-cpp.rst

```diff
@@ -0,0 +1,25 @@
+.. spelling::
+
+    opentracing
+    cpp
+
+.. index:: frameworks ; opentracing-cpp
+
+.. _pkg.opentracing-cpp:
+
+opentracing-cpp
+===============
+
+-  `Official <https://github.com/opentracing/opentracing-cpp>`__
+-  `Hunterized <https://github.com/hunter-packages/opentracing-cpp>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/opentracing-cpp/CMakeLists.txt>`__
+-  Added by `Isaac Hier <https://github.com/isaachier>`__ (`pr-1070 <https://github.com/ruslo/hunter/pull/1070>`__)
```

Comment:
@isaachier Do you think in general we need pull request number and link here? I've just checked that we can do few clicks from documentation to find out origin: `Edit on GitHub` -> `History` -> First commit -> Link to pull request. It was useful when this page was part of the GitHub wiki which doesn't belong to main repository. I guess it's not the case right now.

Reply:
From https://github.com/ruslo/hunter/pull/1072#discussion_r142896621:

> I think we should add the PR, then one can see since when the project is part of hunter (with relase and date)

@NeroBurner It's still possible to reach everything with few clicks (see above). So the question is about "few clicks" vs. making commit and pull **after** PR created (which is a little bit clumsy):
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests

Reply:
I find it can be sort of a pain because of the inherent race condition in determining the pull request number, but it isn't too bad either way.